grammars/asciidoc.cson

```diff
@@ -48,7 +48,7 @@
   # ========================
   #
   {
-    'begin': '^\\.(\\w[\\w ]*\\w)'
+    'begin': '^\\.(\\w.*\\w)'
```

Comment:
The plugin began as an MVP and, in the essence of time, the original line
was overly simplified. Now we just need to make it accurate.